app/ai/voice/agents/breeze_buddy/template/ui_catalog.py

```
@@ -205,6 +205,56 @@ class Buttons(_CatalogBase):
+        description=(
+            "Payload sent to the agent. Falls back to ``label`` when absent — "
+            "lets the display text differ from the agent instruction."
+        ),
+    )
+
+
+class QuickReplies(_CatalogBase):
+    """Turn-scoped quick-reply row — the LLM-driven equivalent of startup
+    chicklets.
+
+    Emit at the end of any assistant turn where you want to suggest 2–5
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)